### PR TITLE
Update configuring-alerts.md

### PR DESCRIPTION
**Pull request by kalevivek11**
formatted the contenet

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by Mariangela — CHANGES_REQUESTED** [review]
@ruks , can you please test this and confirm the change from a technical point of view please.

File: en/docs/learn/analytics/configuring-alerts.md [hunk 1]
```diff
@@ -225,4 +226,4 @@ Follow the instructions below to manage alert types via the Developer Portal:
         </body>
     </html>
 
-Immediately after the request count of the API exceeds the above-defined request count per minute an alert gets triggered. These alerts could be treated as indications of possible high traffic, suspicious activity, possible malfunction of the client application etc.
\ No newline at end of file
+Immediately after the request count of the API exceeds the above-defined request count per minute an alert gets triggered. These alerts could be treated as indications of possible high traffic, suspicious activity, possible malfunction of the client application etc.
```
Comment (ruks) [inline en/docs/learn/analytics/configuring-alerts.md:229]: Have you done any changes here? If not can you please revert this changes?

File: en/docs/learn/analytics/configuring-alerts.md [hunk 2]
```diff
@@ -68,14 +68,15 @@ The users of your APIs can subscribe to analytics-related alerts from the API Pu
...
-                mail.store.protocol: pop3
+                    username: <senders-email-username>
+                    address: <senders-email-address>
+                    password: <senders-email-password>
+                    host: <senders-email-host>
+                    port: 25
+                    ssl.enable : false
+                    auth : false
+                    mail.store.protocol: pop3
+                    mail.smtp.starttls.enable: true
```
Comment (ruks) [inline en/docs/learn/analytics/configuring-alerts.md:79]: Can you maintain the original indentation and update only the required line?

**Review by ruks — CHANGES_REQUESTED** [review]

File: en/docs/learn/analytics/configuring-alerts.md (hunk 2 above)
Comment (kalevivek11) [reply en/docs/learn/analytics/configuring-alerts.md:79]: with original indentation there was an error in logs, indentation is definitely required after this configuration only it started working
I will appreciate if could merger

Comment (kalevivek11) [reply en/docs/learn/analytics/configuring-alerts.md:79]: any update?

Comment (kalevivek11) [reply en/docs/learn/analytics/configuring-alerts.md:79]: any update, please?

**Review by ruks — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]
